### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shivamragnar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,3 +56,51 @@ How to contribute ?
...
+Folder structure
+----------------
+
+```
+.
+├── LICENSE
+├── README.md
+├── build
+│   ├── favicon.ico
+│   ├── logo192.png
```
Comment (TheDataAreClean) [inline README.md:69]: don't need to name last level files.

File: README.md [hunk 2]
```diff
@@ -64,3 +56,51 @@ How to contribute ?
...
+│   ├── reducers
+│   ├── scss
+│   ├── serviceWorker.js
+│   ├── store.js
+│   └── utils
+└── yarn.lock
+```
+License
+-------
+The code is licensed under MIT License while the contents inside the content folder are licensed under CC-BY-4.0.
```
Comment (TheDataAreClean) [inline README.md:106]: Link to CCBY page: https://creativecommons.org/licenses/by/4.0/

**Review by TheDataAreClean — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shivamragnar) [reply README.md]: Made a few changes in the PR, you can have a look @TheDataAreClean.

File: README.md (hunk 2 above)
Comment (shivamragnar) [reply README.md]: Yo done

**Review by TheDataAreClean — APPROVED** [review]

**Pull request merged**